lib/arjdbc/abstract/connection_management.rb

```diff
@@ -2,19 +2,33 @@ module ArJdbc
   module Abstract
     module ConnectionManagement
 
+      # @override
       def active?
+        return unless @connection
```

Comment:
it isn't 100% necessary - its useful to be able to distinguish a non-connected state
... its pbly a very rare state at this point (if ever happening)

Reply:
Yeah, seemed like something that shouldn't happen in regular use, but just so it has a consistent return type thought it would be good to return a boolean. Isn't there a different method to call to check if you are connected?

Reply:
its kind of expected to do both - believe AR's pool is using it.
starting to recall why it was done that way (this was changeg while cp-ing from `JdbcAdapter`) - that case was useful to distinguish whether to attempt a reconnect or not.
also Rails's default implementation returns `nil`. 
Rails isn't from the `strict?` true/false side is it? :wink:

Reply:
lol works for me then